views/Comments/Comments.jsx

```diff
@@ -250,7 +250,7 @@ export default class CommentsView extends React.Component {
                         data={comment}
                         updateComment={this.updateResource}
                         enableEditing={enableCommentEditing}
-                        enableDeleting={loggedInUser.isAdmin}
+                        enableDeleting={loggedInUser.role.includes("admin")}
```

Comment:
Here and line 237 we should add some error checking to ensure user objects actually have the `role` field (don't think existing objects get updated with the model in the twin server PR). If the object doesn't have the field `enableDeleting` should be false.

Reply:
Might be worth making the role field on the model required (with a default value of empty string), then writing a script to go and update all the past documents -- Just to avoid having to check for the field everywhere we use it.